app/views/admin/index.html.erb

```diff
@@ -21,6 +21,7 @@
             <td><%= request.origin_location %></td>
             <td><%= request.destination %></td>
             <td><%= request.item_id %></td>
+            <td><%= request.item_title %></td>
```

Comment:
According to the mockups (page 37) this should be a link to the SearchWorks record.

![sw-title](https://cloud.githubusercontent.com/assets/96776/7402961/cc532bbe-ee85-11e4-848c-ed8048e5c5a1.png)